telemetry/server.go

```diff
@@ -40,13 +41,11 @@ func NewServer(db *sql.DB, logger *zap.Logger) *Server {
 	}
 
 	server.Router.HandleFunc("/protocol-stats", server.createProtocolStats).Methods("POST")
-	server.Router.HandleFunc("/received-messages", server.createReceivedMessages).Methods("POST")
-	server.Router.HandleFunc("/waku-metrics", server.createWakuTelemetry).Methods("POST")
-	server.Router.HandleFunc("/received-envelope", server.createReceivedEnvelope).Methods("POST")
-	server.Router.HandleFunc("/sent-envelope", server.createSentEnvelope).Methods("POST")
 	server.Router.HandleFunc("/update-envelope", server.updateEnvelope).Methods("POST")
```

Comment:
Cleaning up these endpoints mean that this is a breaking change, since clients out there will no longer be able to push metrics. Do we want to allow backwards compatibility for telemetry?

Reply:
Yeah, good question. I was not sure if they are still actively used. Happy to bring them back if you think it would break things

Reply:
Not sure tbh. 
I think it would depend whether these changes will be available in status release 2.30 or not, and if not, if we care about the telemetry data from 2.30 users